src/componnets/CardCart.js

```diff
@@ -0,0 +1,62 @@
+import { useState } from "react";
+import classes from "./Card.module.css";
+
+const CardCart = ({ productData, onDelete, onEditProductAmount }) => {
+  const [amount, setAmount] = useState(productData.amount);
```

Comment:
Is this extra state necessary? Can we not just use the amount from the product data prop. It is updated in parallel I would think.

Reply:
It was not necessary; it was a mistake.